### PR TITLE
fix highlight menu

### PR DESCRIPTION
**Pull request by alavieille**
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2195
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/368

### CODE REVIEW

File: BackofficeBundle/Resources/public/template/Menu/menuView._tpl.html [hunk 1]
```diff
@@ -14,35 +14,35 @@
     <% } %>
     <% if (user.access_section.contribution) { %>
         <li>
-            <a href="#menu-contribution" role="tab" data-toggle="tab" id="navigation-contribution">
```
Comment (itkg-nanne) [inline BackofficeBundle/Resources/public/template/Menu/menuView._tpl.html]: Why remove this? Useless here, i'm ok. But open if an integrator want to highlight just this menu enter.

Comment (alavieille) [inline BackofficeBundle/Resources/public/template/Menu/menuView._tpl.html]: i can use href like a selector

**Comment by itkg-nanne** [comment]
:+1:

File: BackofficeBundle/Resources/public/template/Menu/menuView._tpl.html (hunk 1 above)
Comment (itkg-nanne) [inline BackofficeBundle/Resources/public/template/Menu/menuView._tpl.html]: href will work as a css class not as an id.

**Pull request merged**